schemas/Claim.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Claim.json",
+    "title": "Claim",
+    "description": "A claim for a future economic event(s) in reciprocity for an economic event that already occurred. For example, a claim for payment for goods received.",
+    "type": "object",
+    "properties": {
+        "action": {
+            "description": "Relates a process input or output to a verb, such as consume, produce, work, improve, etc.",
```

Comment:
Relates a process input or output to a verb?
S/b 
"Relates a claim to a verb"?

But should a claim even have an action? If so, why?